docs/dev/Lua API.rst

```diff
@@ -1599,50 +1602,41 @@ Units module
 
   Returns a list of units (possibly empty) assigned to the given noble role.
 
-* ``dfhack.units.getCitizens([exclude_residents, [include_insane]])``
+* ``dfhack.units.getCitizens([exclude_residents, include_insane])``
```

Comment:
should be `dfhack.units.getCitizens([exclude_residents[, include_insane]])` since the function can take 0, 1, or 2 parameters. `dfhack.units.getCitizens([exclude_residents, include_insane])` means it takes 0 or 2, but not 1.